docs/pages/notifications.md

```diff
@@ -10,7 +10,81 @@ description: Notifications alert users to the state of a form or page. In forms,
   can call out important information about the content (including if the content
   is still loading).
 variation_groups:
-  - variation_group_name: Types
+  - variation_group_name: Information (base) notification
+    variations:
+      - variation_code_snippet: >-
+          <div class="m-notification
+                      m-notification__visible">
+              {% include icons/information-round.svg %}
+              <div class="m-notification_content">
+                  <div class="h4 m-notification_message">A default notification</div>
```

Comment:
@anselmbradford Should we change the messages in these examples to use "information" instead of "default" too?